src/components/VideoContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import { useParams } from 'react-router-dom';
+import { StyledVideoContainer } from './styles/VideoContainer.styled';
+import useFetch from '../hooks/useFetch';
+import Alert from 'react-bootstrap/Alert';
+import Spinner from './Spinner';
+import ReactPlayer from 'react-player';
+
+const VideoContainer = () => {
+  const youtubeUrl = 'https://www.youtube.com/watch?v=';
```

Comment:
this and the first section of the api `https://youtube.googleapis.com/youtube/v3/` might be better stored on a constants files